projects/fal/src/fal/exceptions/handlers.py

```diff
@@ -37,7 +37,11 @@ def should_handle(self, exception: Exception) -> bool:
         return isinstance(exception, FalServerlessException)
 
     def handle(self, exception: FalServerlessException):
-        console.print(f"{CROSS_ICON} {exception.message}")
+        msg = f"{CROSS_ICON} {exception.message}"
+        cause = exception.__cause__
+        if cause is not None:
+            msg += f": {str(cause)}"
+        console.print(msg)
         if exception.hint is not None:
```

Comment:
Having hint be part of `__str__` for this exception is a bit odd, esp having newlines there. I'll leave this as is for now, but will revisit that later.